README.md

```diff
@@ -18,6 +18,8 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 1. [CatalogReaderExample](CatalogReaderExample) is a sample showing how to read the NuGet API's catalog resource.
 
+1. [PackageDownloadsExample](PackageDownloadsExample) shows how to get a package's download counts.
```

Comment:
...shows how to get a package's and all its versions' download counts.

Reply:
Thanks for the feedback! I'd prefer to be succinct and I don't think this clarification adds too much value. Please let me know if you feel strongly about this